geometry/calipers_box.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace m2
+{
+// When a size of a convex hull over a set of |points| is less than 4,
```

Comment:
s/a size/the size/
s/a convex hull/the convex hull/

s/a set/the set/
or
s/a set of //

Reply:
Done